src/sentry/static/sentry/app/views/organizationGroupDetails/groupMerged/mergedList.tsx

```diff
@@ -1,37 +1,53 @@
 import React from 'react';
 import styled from '@emotion/styled';
-import PropTypes from 'prop-types';
 
 import EmptyStateWarning from 'app/components/emptyStateWarning';
 import Pagination from 'app/components/pagination';
 import {Panel} from 'app/components/panels';
 import QueryCount from 'app/components/queryCount';
 import {t} from 'app/locale';
-import SentryTypes from 'app/sentryTypes';
+import {Fingerprint} from 'app/stores/groupingStore';
+import {Project} from 'app/types';
 
 import MergedItem from './mergedItem';
 import MergedToolbar from './mergedToolbar';
 
-class MergedList extends React.Component {
-  static propTypes = {
-    onUnmerge: PropTypes.func.isRequired,
-    onToggleCollapse: PropTypes.func.isRequired,
-    items: PropTypes.arrayOf(SentryTypes.Event),
-    pageLinks: PropTypes.string,
-    orgId: PropTypes.string.isRequired,
-    project: SentryTypes.Project.isRequired,
+type Props = {
+  // From GroupMergedView -> handleUnmerge
+  onUnmerge: () => {
+    groupId: string;
+    loadingMessage: string;
+    successMessage: string;
+    errorMessage: string;
   };
+  // From GroupingActions.toggleCollapseFingerprints
+  onToggleCollapse: () => void;
+  fingerprints?: Fingerprint[];
+  pageLinks?: string;
```

Comment:
We need to automate this (and I usually don't point it out in reviews), but we try to have optional types sorted to the bottom